src/SeleniumLibrary/keywords/webdrivertools/webdrivertools.py

```diff
@@ -161,7 +161,7 @@ def create_headless_chrome(
     ):
         if not options:
             options = webdriver.ChromeOptions()
-        options.headless = True
+        options.add_argument('--headless')
```

Comment:
For Chrome we should be using `--headless=new`. [[Reference](https://www.selenium.dev/blog/2023/headless-is-going-away/)]